python_modules/libraries/dagster-dbt/dagster_dbt/cli/resources.py

```diff
@@ -172,6 +172,21 @@ def seed(
         """
         return self.cli("seed", show=show, **kwargs)
 
+    def freshness(
+        self, select: List[str] = None, **kwargs
+    ) -> DbtCliOutput:
+        """
+        Run the ``source snapshot-freshness`` command on a dbt project. kwargs are passed in as additional parameters.
+
+        Args:
+            select (List[str], optional): the sources to include in the run.
+
+        Returns:
+            DbtCliOutput: An instance of :class:`DbtCliOutput<dagster_dbt.DbtCliOutput>` containing
+                parsed log output as well as the contents of run_results.json (if applicable).
+        """
+        return self.cli("source snapshot-freshness", **kwargs)
```

Comment:
I think you want to have this line be `return self.cli("source snapshot-freshness", select=select, **kwargs)` (right now it will discard the select parameter).

Reply:
sounds good! i noticed that in dbt seed (right above where i added freshness) it also takes a select argument, but doesnt specify it in the return like i had here 
`return self.cli("seed", show=show, **kwargs)`

would you like me to add it in there as well since im already updating this file?

Reply:
ah good catch -- yep select and exclude should both be added there, thanks for spotting that!

Reply:
just added!